docs/source/cli.rst

```diff
@@ -51,6 +51,15 @@ You can provision your device without any extra apps by using the ``kasa wifi``
 
 As with all other commands, you can also pass ``--help`` to both ``join`` and ``scan`` commands to see the available options.
 
+.. note::
+
+    For devices requiring authentication, you need to pass valid credentials to provision with the device.
+    At the moment, the only confirmed working combination is username ``test@tp-link.net`` and password ``test``, which has been confirmed to work on some Tapo devices.
```

Comment:
Can we delete this section in line with not exposing default passwords and it happens automatically now.

Reply:
Revised the whole note, hope it's more informative now.